pkg/rpc/server/ospkg/server.go

```diff
@@ -39,7 +39,7 @@ func (s *Server) Detect(_ context.Context, req *proto.OSDetectRequest) (res *pro
 		return nil, err
 	}
 
-	s.vulnClient.FillInfo(vulns, "", false)
+	s.vulnClient.FillInfo(vulns, req.OsFamily, false)
```

Comment:
I'm not sure if this is correct since we have to maintain `Detect` for backwards compatibility.